### PR TITLE
Create IBC light clients in IBC scenario

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: ibc.ts/src/scenario/index.ts [hunk 1]
```diff
@@ -1,7 +1,87 @@
...
+    debug("Create light client");
+    const counterpartyBlockNumber = await counterpartyChain.latestHeight();
+    const blockNumber = await chain.latestHeight();
+    debug(`height is ${counterpartyBlockNumber}`);
+    const counterpartyRawHeader = await counterpartyChain.queryChainHeader(
+        counterpartyBlockNumber
+    );
+    debug(`rawHeader is ${counterpartyRawHeader}`);
+
+    assert(counterpartyRawHeader, "headet should not empty");
```
Comment (junha1) [inline ibc.ts/src/scenario/index.ts]: "header should not be empty"

Comment (majecty) [reply ibc.ts/src/scenario/index.ts]: @junha1 I fixed it.

Comment (junha1) [reply ibc.ts/src/scenario/index.ts]: @majecty  not 'be'?

Comment (majecty) [reply ibc.ts/src/scenario/index.ts]: @junha1 I fixed it.

**Pull request merged**